apps/heft/src/plugins/SassTypingsPlugin/SassTypingsGenerator.ts

```diff
@@ -35,7 +35,12 @@ export interface ISassConfiguration {
    * The paths should be relative to the project root.
    * Defaults to ["node_modules", "src"]
    */
-  includePaths?: string[];
+  importPaths?: string[];
+
+  /**
+   * A list of file paths relative to the "src" folder that should be excluded from typings generation.
+   */
```

Comment:
Probably better to use excludePaths to align with importPaths?

Reply:
This has different behavior than `importPaths`. I wanted to changes the name of that one to make it clearly different. Maybe `importPaths` isn't the right name.